### ISSUE TITLE
Warning: preg_match(): Unknown modifier '^'

### ISSUE THREAD

**Issue by Soullivaneuh** [browscap/browscap-php#125]
Just updated browscap-php, got this:

```
Warning: preg_match(): Unknown modifier '^'
in vendor/browscap/browscap-php/src/phpbrowscap/Browscap.php at line 343
```

browscap/browscap-php updated from 2.0.5 to 2.1.0.

**Comment by mrardon** [comment]
:+1:

**Comment by asgrim** [comment]
Does this affect any UA, or just particular ones please?

**Comment by mrardon** [comment]
I am getting it in latest version of Chrome for Mac on localhost. I am running PHP 5.5.30. Seems to have happened around here: https://github.com/browscap/browscap-php/commit/91f2389d72e4e40a6184c8d9f6065a0c079ef45a

**Comment by asgrim** [comment]
Closed in PR #126
